New

gr5620 opened the pull request:
still making changes but need to see the website before I try to make it more mobile friendly.

gr5620 committed:
Create icon-automatic-01.png

gr5620 committed:
Create icon-support-white-back-01.png

gr5620 committed:
Create icononlinetraining-01.png

gr5620 committed:
Update about.html

fixed layout

gr5620 committed:
Update style.css

add styles from pages

gr5620 committed:
Update issues.html

removed style code from page

gr5620 committed:
Update EDU.html

removed style code from page

gr5620 committed:
Update index.html

removed style code from page. Plus added new content

gr5620 committed:
Update index.html

gr5620 committed:
Update custom.html

fixed link

axamei commented on EDU.html:
```diff
@@ -54,7 +54,7 @@
       <img class="scale-with-grid" style="max-width: 100%" align="right" src="images/icon-engage-01.png" width="180"
         height="180">
-      <h4 style="color: #fff; font-size: 32px;">A Thriving Library Begins with your People <p></p>
+      <h5>A Thriving Library Begins with your People <p></p>
```
You can't skip heading levels. Adjust the size with CSS

gr5620 committed:
Update EDU.html

moved edu img style to CSS

gr5620 committed:
Update style.css

moved edu img style to CSS

gr5620 committed:
Update about.html

fixed logo

gr5620 committed:
Update custom.html

fixed logo

gr5620 committed:
Update EDU.html

fixed logo

gr5620 committed:
Update form.html

fixed logo

gr5620 committed:
Update index.html

fixed logo

gr5620 committed:
Update issues.html

fixed logo

gr5620 committed:
Update Services.html

fixed logo

gr5620 committed:
Update style.css

fixed logo

gr5620 committed:
Update about.html

removed style code

gr5620 committed:
Update custom.html

removed style code

gr5620 committed:
Update EDU.html

removed style code

gr5620 committed:
Update index.html

removed style code

gr5620 committed:
Update Services.html

gr5620 committed:
Update issues.html

removed style code

gr5620 committed:
Update form.html

removed style code

gr5620 committed:
Update style.css

updates style code

gr5620 committed:
Update form.html

gr5620 committed:
Update style.css

gr5620 committed:
Update index.html

gr5620 committed:
Update form.html

gr5620 committed:
Update icons-02-apple.png

gr5620 committed:
Update Services.html

gr5620 committed:
Update issues.html

gr5620 committed:
Update EDU.html

gr5620 committed:
Update custom.html

gr5620 committed:
Update about.html

gr5620 committed:
Update form.html

gr5620 committed:
Update style.css

gr5620 committed:
Update index.html

ran out of time tonight. Will work on changing buttons to lists tomorrow evening.

gr5620 committed:
Update index.html

gr5620 committed:
Update Services.html

gr5620 committed:
Update style.css

gr5620 committed:
Update style.css

gr5620 committed:
Update about.html

gr5620 committed:
Update EDU.html

gr5620 committed:
Update custom.html

gr5620 committed:
Update form.html

gr5620 committed:
Update index.html

gr5620 committed:
Update issues.html

gr5620 committed:
Update Services.html

gr5620 committed:
Update style.css

gr5620 committed:
Update EDU.html

gr5620 committed:
Create guide-to-degrees.png

gr5620 committed:
Create Support-Center-sm.png

gr5620 committed:
Update index.html

gr5620 committed:
Update about.html

gr5620 committed:
Update custom.html

gr5620 committed:
Update EDU.html

gr5620 committed:
Update form.html

gr5620 committed:
Update Services.html

gr5620 committed:
Update issues.html

gr5620 committed:
Update style.css

gr5620 commented:
made changes to make more site cohesive and mobile friendly.
